webpack/scenes/Subscriptions/SubscriptionActions.js

```diff
@@ -106,7 +106,7 @@ export const resetTasks = () => (dispatch) => {
 
 export const handleTask = task => async (dispatch, getState) => {
   if (selectIsPollingTask(getState(), SUBSCRIPTIONS)) {
-    if (!task.pending) {
+    if (!task.pending && task.result !== 'pending') {
```

Comment:
`task.pending` will be true if `task.state === 'pending'`.  So I'm not sure why we'd ever have a state of "pending" and not a result of "pending," but figured it can't hurt to address that case..